scripts/Muon/GUI/Common/list_selector/list_selector_view.py

```diff
@@ -0,0 +1,91 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from qtpy import QtWidgets
+
+from qtpy.QtCore import Qt
+from Muon.GUI.Common.list_selector.TableWidgetDragRows import TableWidgetDragRows
+from mantidqt.utils.qt import load_ui
+from Muon.GUI.Common.utilities import table_utils
+
+ui_list_selector, _ = load_ui(__file__, "list_selector.ui")
+
+
+class ListSelectorView(QtWidgets.QWidget, ui_list_selector):
+    def __init__(self, parent_widget=None):
+        super(QtWidgets.QWidget, self).__init__(parent=parent_widget)
+        self.setupUi(self)
+
+        self.item_table_widget = TableWidgetDragRows()
+
+        self.item_table_widget.setColumnCount(2)
+        self.item_table_widget.setColumnWidth(0, 350)
+        self.item_table_widget.verticalHeader().setVisible(False)
+        self.item_table_widget.horizontalHeader().setStretchLastSection(True)
+        self.item_table_widget.horizontalHeader().setVisible(False)
+
+        self.list_layout.addWidget(self.item_table_widget)
+        self.list_layout.setContentsMargins(0, 0, 0, 0)
+
+        self._item_selection_changed_action = lambda a, b: 0
+        self.item_table_widget.cellChanged.connect(self.handle_cell_changed_signal)
+
+    def addItems(self, item_list):
+        """
+        Adds all the items in item list to the table
+        :param itemList: A list of tuples (item_name, check_state, enabled)
+        """
+        self.item_table_widget.blockSignals(True)
+        for index, row in enumerate(item_list):
+            insertion_index = self.item_table_widget.rowCount()
+            self.item_table_widget.setRowCount(insertion_index + 1)
+            table_utils.setRowName(self.item_table_widget, insertion_index, row[0])
```

Comment:
for the column numbers could we have variables names at the top instead and then use them